.github/workflows/blueprint.yml

```
@@ -79,10 +79,17 @@ jobs:
             !.lake/build/doc/declarations/declaration-data-equational_theories*
           key: LakeBuild-${{ runner.os }}-${{ hashFiles('lean-toolchain') }}-${{ hashFiles('lake-manifest.json') }}-${{ github.sha }}
           restore-keys: LakeBuild-
-
+      
       - name: Build project
         run: ~/.elan/bin/lake build equational_theories
 
+      - name: "Check with lean4checker"
```

I would keep homogenous style.

```suggestion
      - name: Check with lean4checker
```

---

It was waiting for my review approval